adapters/base/src/main/java/org/apache/cassandra/sidecar/adapters/base/TokenRangeReplicas.java

```diff
@@ -219,7 +219,7 @@ public static List<TokenRangeReplicas> normalize(List<TokenRangeReplicas> ranges
 
         if (ranges.stream().noneMatch(r -> r.partitioner.minToken.compareTo(r.start()) == 0))
         {
-            LOGGER.warn("{} based minToken does not exist in the token ranges", Partitioner.class.getName());
+            LOGGER.warn("Partitioner based minToken does not exist in the token ranges");
```

Comment:
it seems that the intent of the logger was to log the partitioner name ? should we make the method non-static so we have access to the configured `partitioner`?

```
LOGGER.warn("{} based minToken does not exist in the token ranges", partitioner.name());
```

Reply:
ah, it looks like this method is called from other places as well..

Reply:
We could extract a single range to get the partitioner to log here. I'll make that change.